docs/tsc/meetings/2020-04-06.md

```diff
@@ -0,0 +1,70 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+April 6, 2020
+
+Host: Michael Dolan
+
+Rotating Secretary: Doug Walker
+
+Attendees:
+  * [X] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [X] Sean Cooper (_TSC_) - DNEG
+  * [X] Michael Dolan (_TSC Chair_) - Epic Games
+  * [X] Larry Gritz (_TSC_) - Sony Pictures Imageworks
+  * [X] Patrick Hodoul (_TSC_) - Autodesk
+  * [ ] John Mertic - Academy Software Foundation / Linux Foundation
+  * [ ] Carl Rand (_TSC_) - Weta Digital
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Bernard Lefebvre - Autodesk
+  * [X] Mei Chu - Sony Pictures Imageworks
+  * [X] Carol Payne - Netflix
+  * [X] Mitch Prater - Laika
+
+# **OCIO TSC Meeting Notes**
+
+* Google Summer of Code
+    - Larry: Need to discuss the OCIO proposals and rank them.  Proposals from all the ASWF
+    projects will get merged into an overall list that we will use once we find out from
+    Google how many slots we will get.
+
+* Moving CI to GitHub Actions
+    - Michael: At the ASWF CI meeting last week there was a good discussion on getting the
+    GPU CI going.  JF and Andy are going to help with that.  For various reasons, we first 
+    need to move our CI from Azure to GH Actions.  I volunteered to do that over the coming
+    weeks.  Do we want to make any platform/compiler changes to our matrix while doing that?
+    - Patrick: Let's start by just moving the current configuration over.
```

Comment:
Please add something like
`Patrick: As Python support is important for OCIOv2, let first complete/merge the Python Bindings pull request if agreement so others could jump to add unit tests, documentations, etc.`